<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_11">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget_5">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Gfx</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea_2">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>545</width>
                <height>587</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0">
                <widget class="QLabel" name="label_img">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Hex</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea_3">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>545</width>
                <height>587</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_13">
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>Auto</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>545</width>
                <height>587</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QMdiArea" name="mdiArea"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QGroupBox" name="groupBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Format</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QComboBox" name="comb_format">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>YUYV</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UYVY</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>YVYU</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>VYUY</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>BGR3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>RGB3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>XR24</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>RGBP</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Size</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLineEdit" name="LineEdit_width">
                 <property name="text">
                  <string>400</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>x</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="LineEdit_height">
                 <property name="text">
                  <string>400</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="apply_button">
               <property name="text">
                <string>Apply</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
          <zorder></zorder>
         </widget>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Endianess</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="radiobutton_be">
               <property name="text">
                <string>BE</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radiobutton_le">
               <property name="text">
                <string>LE</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QGroupBox" name="group_rgb">
           <property name="title">
            <string>Channels</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QCheckBox" name="checkbox_r">
                <property name="text">
                 <string>R</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkbox_g">
                <property name="text">
                 <string>G</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkbox_b">
                <property name="text">
                 <string>B</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="group_yuv">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QCheckBox" name="checkbox_y">
                <property name="text">
                 <string>Y</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <property name="tristate">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkbox_u">
                <property name="text">
                 <string>U</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkbox_v">
                <property name="text">
                 <string>V</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QCheckBox" name="checkbox__a_yuv">
                <property name="text">
                 <string>A</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkbox_swap">
                <property name="text">
                 <string>Swap UV</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="auto_btn">
             <property name="text">
              <string>AutoDetection</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="hex_btn">
             <property name="text">
              <string>Hex button</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_PValues">
           <attribute name="title">
            <string>Pixel values</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="txt_PixelValue_X">
                  <property name="text">
                   <string>X:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="UserInput_PixelValue_X"/>
                </item>
                <item>
                 <widget class="QLabel" name="txt_PixelValue_Y">
                  <property name="text">
                   <string>Y:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="UserInput_PixelValue_Y"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="2" column="1">
                 <widget class="QLineEdit" name="UserInput_PixelValue_B"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="txt_PixelValue_G">
                  <property name="text">
                   <string>G:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="txt_PixelValue_R">
                  <property name="text">
                   <string>R:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="UserInput_PixelValue_R"/>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="txt_PixelValue_B">
                  <property name="text">
                   <string>B:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="UserInput_PixelValue_G"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Ifo.</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_14">
            <item row="1" column="0">
             <widget class="QScrollArea" name="scrollArea_4">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_4">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>346</width>
                 <height>139</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout_15">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_info">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QProgressBar" name="pg">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
   </widget>
   <widget class="QMenu" name="menuImages">
    <property name="title">
     <string>info</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuImages"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-color: rgb(7, 148, 229);

</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave_As"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="actionHelp"/>
   <addaction name="actionAbout"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icon/open.qrc">
     <normaloff>:/open/open.qrc</normaloff>
     <normalon>:/open/open.png</normalon>
     <activeoff>:/open/open.png</activeoff>
     <activeon>:/open/open.png</activeon>
     <selectedoff>:/open/open.png</selectedoff>
     <selectedon>:/open/open.png</selectedon>:/open/open.qrc</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="icon/save.qrc">
     <normaloff>:/save/save.png</normaloff>:/save/save.png</iconset>
   </property>
   <property name="text">
    <string>Save_As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icon/exit.qrc">
     <normaloff>:/exit/exit.png</normaloff>:/exit/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset resource="icon/zoomin.qrc">
     <normaloff>:/zoomin/zoomin.png</normaloff>:/zoomin/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>Zoom_in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset resource="icon/zoomout.qrc">
     <normaloff>:/zoomout/zoomout.png</normaloff>:/zoomout/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>Zoom_out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="icon/help.qrc">
     <normaloff>:/help/help.png</normaloff>:/help/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icon/about.qrc">
     <normaloff>:/about/about.png</normaloff>:/about/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon/about.qrc"/>
  <include location="icon/exit.qrc"/>
  <include location="icon/help.qrc"/>
  <include location="icon/open.qrc"/>
  <include location="icon/save.qrc"/>
  <include location="icon/zoomin.qrc"/>
  <include location="icon/zoomout.qrc"/>
 </resources>
 <connections/>
</ui>
